src/clr-angular/utils/i18n/common-strings.service.ts

```diff
@@ -29,6 +29,8 @@ export class ClrCommonStringsService implements ClrCommonStrings {
   danger = 'Error';
   rowActions = 'Available actions';
   pickColumns = 'Show or hide columns';
+  min = 'Min';
```

Comment:
I think we need to make these more clear `Minimum value` and `Maximum value`.

Reply:
At the current input widths these placeholders would get cut off.  Is that OK?

Reply:
`Min value` and `Max value` mostly fit if that works:

![image](https://user-images.githubusercontent.com/7132414/57272909-830bf300-704a-11e9-8e11-61a982ab2c44.png)

Reply:
I can agree with that. It might also depend on the browser.